src/Source.jl

```diff
@@ -320,13 +320,20 @@ Data.streamtype(::Type{ODBC.Source}, ::Type{Data.Column}) = true
 Data.streamtype(::Type{ODBC.Source}, ::Type{Data.Field}) = true
 
 function Data.streamfrom(source::ODBC.Source, ::Type{Data.Field}, ::Type{Union{T, Missing}}, row, col) where {T}
-    val = source.columns[col][row - source.rowoffset]::Union{T, Missing}
+    val = if isempty(source.columns[col])
```

Comment:
I'm having a hard time walking thru the logic of the changes right here specifically; can you help me understand why this needs ot change?

Reply:
@quinnj Sure, this particular case happens the first time `streamfrom` gets called, `source.columns[col]` is just an empty array (nothing is fetched), which causes this error: 

```julia
BoundsError: attempt to access 0-element Array{Union{Int64, Missings.Missing},1} at index [1]
```
Full stacktrace here: #196 

Let me know if you have any ideas of a better way to fix this.